Makefile

```diff
@@ -21,6 +21,16 @@ MELANGE_OPTS += --generate-index false
 MELANGE_OPTS += --pipeline-dir ./pipelines/
 MELANGE_OPTS += ${MELANGE_EXTRA_OPTS}
 
+# These are separate from MELANGE_OPTS because for building we need additional
+# ones that are not defined for tests.
```

Comment:
No, there are additional things that test does not need. For example signing key, or create index, etc. So, if they are passed as is to test, it will fail because those flags are not supported there. I could split these into:
MELANGE_OPTS (shared base options)
MELANGE_BUILD_OPTS
MELANGE_TEST_OPTS if that would make more sense, wdyt?

Reply:
ahh I gotcha. I misread the comment.

this lgtm, I don't think we need to get super dry with shared base options :D